tensorflow/security/fuzzing/consume_leading_digits_fuzz.cc

```diff
@@ -18,21 +18,26 @@ limitations under the License.
 #include "tensorflow/core/platform/str_util.h"
 #include "tensorflow/core/platform/stringpiece.h"
 
+#include <fuzzer/FuzzedDataProvider.h>
+
 // This is a fuzzer for tensorflow::str_util::ConsumeLeadingDigits
 
 namespace {
 
 extern "C" int LLVMFuzzerTestOneInput(const uint8_t *data, size_t size) {
-  uint8_t *byte_data = const_cast<uint8_t *>(data);
-  char *char_data = reinterpret_cast<char *>(byte_data);
-
-  tensorflow::StringPiece sp(char_data, size);
-  tensorflow::uint64 val;
-
-  const bool leading_digits =
-      tensorflow::str_util::ConsumeLeadingDigits(&sp, &val);
-  if (leading_digits) {
-    assert(val >= 0);
+  FuzzedDataProvider fuzzed_data(data, size);
+
+  while (fuzzed_data.remaining_bytes() > 0) {
+    std::string s = fuzzed_data.ConsumeRandomLengthString(5);
```

Comment:
```suggestion
    std::string s = fuzzed_data.ConsumeRandomLengthString(25);
```

Let's use `25` to be able to also generate overflow (`uint64`, `log10(2^64) ~= 64 * 0.3 = 19`, keep a few more chars for random string output)